lib/livebook_web/live/file_select_component.ex

```diff
@@ -87,11 +87,12 @@ defmodule LivebookWeb.FileSelectComponent do
   @impl true
   def render(assigns) do
     ~H"""
-    <div class="h-full flex flex-col">
+    <div id={@id} class="h-full flex flex-col">
       <h2 class="sr-only">File system</h2>
       <div class="flex space-x-3 items-center mb-4">
         <div class="grow flex space-x-1">
           <.file_system_menu_button
+            id={@id}
```

Comment:
Double checking this won't lead to duplicate IDs, as we use `id={@id}` above too.

Reply:
Generally speaking, I would prefer to build the new ID here instead of inside the component. Because `id={@id}` will always make me pause and double check.

Reply:
We pass the base id and use in the component to scope, so no duplicates, but I will move the id generation up so it's not confusing :)